InjectionIII/AppDelegate.swift

```diff
@@ -70,7 +70,8 @@ class AppDelegate : NSObject, NSApplicationDelegate {
                 .contentsOfDirectory(atPath: open.url!.path)
             if let fileList = fileList, let projectFile =
                fileWithExtension("xcworkspace", inFiles: fileList) ??
-               fileWithExtension("xcodeproj", inFiles: fileList),
+               fileWithExtension("xcodeproj", inFiles: fileList) ??
+                fileList.filter({$0 == "Package.swift"}).first,
```

Comment:
Could we use `first(where:)` here?

https://developer.apple.com/documentation/swift/anybidirectionalcollection/2906322-first#

Reply:
Good point, committed. Will merge now. Do you have time to prepare a release candidate?

Reply:
A release candidate is up!

https://github.com/johnno1962/InjectionIII/releases/tag/1.8

Reply:
Thanks @zenangst, I figure we'll release in due course. No hurry for now.